src/analytics/models/graph.py

```diff
@@ -0,0 +1,103 @@
+import app 
+from helpers import mongo_helpers, helpers
```

Comment:
Could we define the schema here and have most of this business logic go under `controllers/Graph.py` ?